sahi/utils/mot.py

```diff
@@ -250,26 +250,31 @@ def _create_frame_symlinks(self, coco_images_dir: str, export_dir: str):
             if Path(mot_frame.file_name).suffix == "":
                 print(f"image file has no suffix, skipping it: '{mot_frame.file_name}'")
                 return
-            elif Path(mot_frame.file_name).suffix not in [".jpg", ".jpeg"]:  # TODO: extend this list
+            elif Path(mot_frame.file_name).suffix not in [".jpg", ".jpeg", ".bmp", ".gif", ".png", ".tiff"]:
                 print(f"image file has incorrect suffix, skipping it: '{mot_frame.file_name}'")
                 return
-            # set coco and mot image paths
-            coco_img_path_tmp = os.path.join(coco_images_dir, mot_frame.file_name)
+            # set source and mot image paths
             suffix = Path(mot_frame.file_name).suffix
 
-            if Path(coco_img_path_tmp).is_file():
-                coco_image_path = os.path.abspath(coco_img_path_tmp)
+            if not mot_frame.file_name:
```

Comment:
verdiğin error ile check uyuşmuyor.

ya error olarak "file_name has to be set" tarzı bir şey demen lazım ya da "if not isinstance(mot_frame.file_name, str):" diyerek check yapman lazım